waku/waku_test.go

```diff
@@ -1491,15 +1533,21 @@ func TestWakuTimeDesyncEnvelopeIgnored(t *testing.T) {
 	}
 	rw1, rw2 := p2p.MsgPipe()
 	defer func() {
-		rw1.Close()
-		rw2.Close()
+		if err := rw1.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
+		if err := rw2.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
 	}()
 	p1 := p2p.NewPeer(enode.ID{1}, "1", []p2p.Cap{{"shh", 6}})
 	p2 := p2p.NewPeer(enode.ID{2}, "2", []p2p.Cap{{"shh", 6}})
 	w1, w2 := New(c, nil), New(c, nil)
 	errc := make(chan error)
 	go func() {
-		w1.HandlePeer(p2, rw2) // nolint: errcheck
+		if err := w1.HandlePeer(p2, rw2); err != nil {
+			t.Errorf("error handling peer, '%s'", err)
+		}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)